eng/common/pipelines/templates/steps/create-apireview.yml

```diff
@@ -24,4 +24,4 @@ steps:
         pwsh: true
         workingDirectory: $(Pipeline.Workspace)
       displayName: Create API Review for ${{ artifact.name}}
-      condition: and(succeededOrFailed(), ne(variables['Skip.CreateApiReview'], 'true') , ne(variables['Build.Reason'],'PullRequest'), eq(variables['System.TeamProject'], 'internal'))
+      condition: and(succeededOrFailed(), ne(variables['Skip.CreateApiReview'], 'true') , ne(variables['Build.Reason'],'PullRequest'), eq(variables['System.TeamProject'], 'internal'), not(endsWith(variables['Build.Repository.Name'], '-pr')))
```

Comment:
Might be worth splitting this condition across multiple lines to help with readability. Something like https://github.com/Azure/azure-sdk-for-cpp/blob/main/eng/pipelines/templates/jobs/ci.tests.yml#L91

Reply:
Thank you. Updated